src/js/components/tour/popover.tsx

```diff
@@ -30,6 +30,7 @@ const TourPopover = ({
         assistiveText={{ icon: i18n.t('Learn More') }}
         iconCategory="utility"
         iconName="info_alt"
+        style={{ color: '#DC72D1' }}
```

Comment:
I wonder if we could do this with a class instead? This is okay, but I think of direct inline `style` attrs as a last-case resort.

Reply:
Right. Added this to the `popover-trigger ` class